### ISSUE TITLE
失效了老哥

### ISSUE THREAD

**Issue by llh1187** [Zachary-YL/Google2#3]
用不了，替换了hosts文件之后上不了。

**Comment by llh1187** [comment]
用得还是校园网哦

**Comment by Zachary-YL** [comment]
The latest host file has been updated named 'host'.

**Comment by llh1187** [comment]
还是不行哦，电脑已经刷新了dns，浏览器也刷新了，用得是校园网，支持ipv6的，请问还有什么我没有做的吗？

**Comment by llh1187** [comment]
我是直接把host里面的内容复制到hosts文件里面的。

**Comment by Zachary-YL** [comment]
At present, due to the interference of the wall to the UDP port 443, some places directly connected to Google search through the QUIC protocol are no longer available; commonly used: Twitter, YouTube, Facebook are also unavailable. Luckily, Google Scholar is available at https://scholar.google.com/.

**Comment by llh1187** [comment]
you are right.

**Comment by llh1187** [comment]
敢问是中国老乡吗？

**Comment by Zachary-YL** [comment]
Yes. But in order to help more people, it is best to use English to communicate.
